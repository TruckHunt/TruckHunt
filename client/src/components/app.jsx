import React from 'react';

export default class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <h1 style={{ textAlign: 'center', fontSize: '12em', color: '#ffffff' }}>Hello World</h1>
    );
  }
}
