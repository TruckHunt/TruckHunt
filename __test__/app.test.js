import { shallow } from 'enzyme';
import React from 'react';
// import renderer from 'react-test-renderer';
import App from '../client/src/components/App';

const sum = (a, b) => (a + b); // example function (import ./server/sum.js)


describe('App Components Test', () => {
  test('adds 1 + 2 to equal 3', () => {
    expect(sum(1, 2)).toBe(3);
  });

  // // snapshot test - to update, run "npm run resnap"
  // // see https://facebook.github.io/jest/docs/snapshot-testing.html#content
  // test('app matches snapshot', () => {
  //   const component = renderer.create(<App />);
  //   const tree = component.toJSON();
  //   expect(tree).toMatchSnapshot();
  // });

  test('app contains header and trucklist', () => {
    const app = shallow(<App />);
    expect(app.text()).toContain('<Header');
    expect(app.text()).toContain('<TruckList');

    // expect(app.text()).toContain('<Header /><Map /><TruckList />');
  });
});
