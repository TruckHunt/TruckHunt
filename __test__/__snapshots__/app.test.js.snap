// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Components Test app matches snapshot 1`] = `
<h1
  style={
    Object {
      "color": "#ffffff",
      "fontSize": "12em",
      "textAlign": "center",
    }
  }
>
  Hello World
</h1>
`;
